plugin/deepness/dialogs/packages_installer/packages_installer_dialog.py

```diff
@@ -286,13 +262,13 @@ def _check_packages_installation_and_log(self) -> bool:
             import_packages()
             raise Exception("Unexpected successful import of packages?!? It failed a moment ago, we shouldn't be here!")
         except Exception as e:
-            msg_base = '<b>Python packages required by the plugin could be loaded due to the following error:</b>'
-            logging.exception(msg_base)
+            msg_base = '<b>Python packages required by the plugin could not be loaded due to the following error:</b>'
+            logging.exception(e)
```

Comment:
Why passing `e` to `logging.exception`? It prints the entire stack and error anyway, the argument is to add a custom message on top, as it was with `msg_base`
@rivi